python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -339,13 +377,16 @@ def measure(self, measRecord: lsst.afw.table.SourceRecord, exposure: afwImage.Ex
                 aperShape = afwGeom.Quadrupole(aperSigma2, aperSigma2, 0.0)
                 fluxResult = measBase.SdssShapeAlgorithm.computeFixedMomentsFlux(convolved.getMaskedImage(),
                                                                                  aperShape, center)
-                fluxScaling = sigma**2/aperSigma2  # Eq. A16 of Kuijken et al. (2015)
-
-                # Copy result to record
-                instFluxKey = measRecord.schema.join(baseName, "instFlux")
-                instFluxErrKey = measRecord.schema.join(baseName, "instFluxErr")
-                measRecord.set(instFluxKey, fluxScaling*fluxResult.instFlux)
-                measRecord.set(instFluxErrKey, fluxScaling*fluxResult.instFluxErr)
+                # Calculate the scale factors by which to scale the fluxes and
+                # their standard errors.
+                fluxScaling = 0.5*sigma**2/aperSigma2  # Eq. A16 of Kuijken et al. (2015)
```

Comment:
I'm looking at Kuijken (2015) equation A16, and I don't see how that maps onto this formula, or which part of the formula this is representing.